tests/unit_tests/cpu/coordinate_cartesian2.cpp

```diff
@@ -25,7 +26,7 @@ using matrix_type = typename matrix_operator::template matrix_type<ROWS, COLS>;
 const scalar isclose{1e-5f};
 
 // This test cartesian2 coordinate
-TEST(coordinate, cartesian2) {
+GTEST_TEST(detray_core, cartesian2) {
```

Comment:
It seems to have replaced all test names with ```detray_core```. Is this what we want?

Reply:
I renamed all the test suites now. That commit should probably be well looked at as I don't _really_ know what I'm doing there